Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Используем официальный образ Python
+FROM python:3.8-slim
```

Comment:
Consider using an ARG for the Python base image version to enhance flexibility and allow easy updates in the future. For example, you can add `ARG PYTHON_VERSION=3.8-slim` and then use `FROM python:${PYTHON_VERSION}`.